integration-tests/drivers/TestDriver063/android/app/build.gradle

```diff
@@ -1,4 +1,5 @@
 apply plugin: "com.android.application"
+apply plugin: 'com.heapanalytics.android'
```

Comment:
So per the comments above, I think RN has avoided using the Android Gradle plugin.